client/src/MessageSend.js

```diff
@@ -0,0 +1,37 @@
+/** @jsx jsx  */
+import React from "react";
+import { jsx } from "@emotion/core";
+
+function MessageSend() {
+  return (
+    <>
+      <div
+        css={{
+          overflowY: "auto",
+          overflowX: "hidden",
+          height: "90%",
+          width: "98%",
+          paddingLeft: "15px",
+          textAlign: "end"
+        }}
+      />
+      <div>
+        <form autocomplete="off">
+          <label for="input-message" aria-label="write a message" />
```

Comment:
This should be `htmlFor` instead of `for`.
```suggestion
          <label htmlFor="input-message" aria-label="write a message" />
```

Reply:
Yeah! I found it then, "for" doesn't work.